public/media_engagement/styles.css

```diff
@@ -1,12 +1,25 @@
-.nav {
-    background: #ffff;
-    width: 100%;
-    max-width: initial;
-    height: 60px;
+.logo {
```

Comment:
I can't find the relevant style, but something you've changed as had the effect of off-setting the cards from the centre of the screen, which doesn't look as neat

Reply:
The body is still using materialize CSS and I have removed that library. I will create another PR for the changes I have made on the body.

Reply:
In that case can you restore that import for now and remove it in the next PR?

Reply:
It will affect the overall apearance of the pages due to conflicts of the classes between bootstrap and materialize css

Reply:
ok fine. This isn't breaking any of the site, but it does worsen the layout of the pages slightly, particularly IMAQAL, so please open an issue to address

Reply:
Opened issue #76